opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx

```diff
@@ -502,6 +503,31 @@ const LeftBar = () => {
           </Security>
         </MenuList>
       </Security>
+      {!platform_whitemark && (
+        <MenuItem
+          dense={true}
+          style={{ marginBottom: bannerHeightNumber }}
+          classes={{
+            root: navOpen ? classes.menuLogoOpen : classes.menuLogo,
+          }}
```

Comment:
This menu item is clickable but does nothing. Open the filigran website in a new window ?

```suggestion
          }}
          onClick={() => window.open('https://filigran.io/', '_blank')}
```

Reply:
@Kedae wdyt?